lxd/operations/operations.go

```diff
@@ -90,6 +92,9 @@ func OperationGetInternal(id string) (*Operation, error) {
 	return op, nil
 }
 
+// CallbackFuncKey keys a callback function to be called after a request is completed.
+const CallbackFuncKey util.ContextKey = "callback"
```

Comment:
Please define a `CtxKey` in lxd/request/const.go to keep this with the others, and the key name should be more specific please.